apps/frontend/src/libs/modules/store/handle-error.middleware.ts

```diff
@@ -0,0 +1,25 @@
+import type { Middleware } from "@reduxjs/toolkit";
+
+import { isRejected } from "@reduxjs/toolkit";
```

Comment:
```suggestion
import { type Middleware } from "@reduxjs/toolkit";

import { isRejected } from "@reduxjs/toolkit";
```
Also combine imports